test/killbill_test_helper.rb

```diff
@@ -22,7 +22,8 @@ def check_no_flash_error
     # Kill Bill specific helpers
     #
 
-    def setup_test_data(nb_configured_tenants, setup_tenant_key_secret)
+    def setup_test_data(nb_configured_tenants, setup_tenant_key_secret, options = {})
+      @options = options
```

Comment:
Suggestion: let's rename the new `@options` (and `options`) to something like `tenant_data`.

`options` has a specific meaning in these files (check `build_options` method): it contains the credentials used in the Ruby client calls. I'm just afraid we get confused one day mixing the two...

Reply:
Thanks @pierre . That makes sense.